source/ScreensaverHero.hx

```diff
@@ -0,0 +1,57 @@
+package;
+
+import flixel.math.FlxPoint;
+import flixel.math.FlxRandom;
+import flixel.util.FlxColor;
+import flixel.FlxSprite;
+
+class ScreensaverHero extends FlxSprite
+{
+	var rand = new FlxRandom();
```

Comment:
Because this variable is defined as an instance-level property, every single one of our ScreensaverHero objects will instantiate their own `FlxRandom` when they're created. 

While FlxRandom isn't crazy heavy to set up, it is some overhead that isn't really necessary. Each FlxRandom is a full random number generator, so just one would be sufficient to get random velocities for all of our `ScreensaverHero` objects.

We'll talk about it more in class, but I recommend using the statically available
`FlxG.random` instead of creating our own. There are reasons why you'd want to make multiple separate FlxRandom instances (like how some games give the player the option of selecting their own random seed for world generation, but other random elements of a playthrough aren't seeded to be identical every time) but if we're not pursuing those sorts of things, we're better off not incurring extra costs by setting up more.